pkg/apis/camel/v1/common_types.go

```diff
@@ -55,6 +55,8 @@ type BuildConfiguration struct {
 	LimitMemory string `property:"limit-memory" json:"limitMemory,omitempty"`
 	// The node selector for the builder pod. Only used for `pod` strategy
 	NodeSelector map[string]string `property:"node-selector" json:"nodeSelector,omitempty"`
+	// Annotation to use for the builder pod. Only user for `pod` strategy
```

Comment:
```suggestion
	// Annotation to use for the builder pod. Only used for `pod` strategy
```

Reply:
Updated with the new commits (changing this triggers many other changes when generating files)